scripts/ai/AIDriveStrategyCourse.lua

```diff
@@ -499,11 +490,12 @@ end
 --- Hold the vehicle (set speed to 0) temporary. This is meant to be used for other vehicles to coordinate movements,
 --- for instance tell a vehicle it should not move as the other vehicle is driving around it.
 ---@param milliseconds number milliseconds to hold
-function AIDriveStrategyCourse:hold(milliseconds)
+function AIDriveStrategyCourse:hold(milliseconds, fuelSaveAllowed)

```

Comment:
Probably missed them, but there aren't a lot of comments on how this refill should work, are we using hold()? If so, can we update the comments to make it clear?